src/scheduler/gc_work.rs

```diff
@@ -855,6 +905,11 @@ pub trait ScanObjectsWork<VM: VMBinding>: GCWork<VM> + Sized {
                         object_tracer,
                     );
                     self.post_scan_object(object);
+
+                    #[cfg(feature = "count_live_bytes_in_gc")]
+                    _worker
+                        .shared
+                        .increase_live_bytes(VM::VMObjectModel::get_current_size(object));
```

Comment:
If we already counted the object sizes in the previous loop, this is unnecessary.

Reply:
Why? I thought the previous loop is for edges, and this loop is for nodes.